packages/monorepo-scripts/src/utils.ts

```diff
@@ -17,4 +22,44 @@ export const utils = {
             cwd,
         });
     },
+    async getUpdatedLernaPackagesAsync(shouldIncludePrivate: boolean): Promise<LernaPackage[]> {
+        const updatedPublicPackages = await this.getLernaUpdatedPackagesAsync(shouldIncludePrivate);
+        const updatedPackageNames = _.map(updatedPublicPackages, pkg => pkg.name);
+
```

Comment:
how come this extra newline didn't get deleted by prettier?

Reply:
Newlines within method bodies are allowed no?